js/apps/account-ui/src/account-security/DeviceActivity.tsx

```diff
@@ -115,17 +114,15 @@ const DeviceActivity = () => {
           </Title>
         </SplitItem>
         <SplitItem>
-          <Tooltip content={t("refreshPage")}>
-            <Button
-              aria-describedby="refresh page"
-              id="refresh-page"
-              variant="link"
-              onClick={() => refresh()}
-              icon={<SyncAltIcon />}
-            >
-              {t("refreshPage")}
-            </Button>
-          </Tooltip>
+          <Button
+            aria-describedby="refresh page"
```

Comment:
I actually cannot find any element that has that has this `id`, and in this case the element is already descriptive, so let's remove this attribute entirely.
```suggestion
```